docs/general/support-contact.md

```diff
@@ -4,30 +4,100 @@ hide:
+| **Level 2** | Information Manager (CVA IM) | - Escalation from Level 1<br>- IM support (e.g., database updates)<br>- Power BI support<br>- Escalation to Level 3  |
+| **Level 3** | Development Team | - Escalation from Level 2<br>- Bug fixes and advanced troubleshooting    |
+<!-- markdownlint-enable -->
+
+________________________________________
+
+### Response Time Expectations
+
+We aim to resolve issues swiftly, based on the urgency of the situation.
+
+<!-- markdownlint-disable -->
+| **Category** | **Response Time** | **Target Resolution Time** | **Description**                                                  |
+|:--------------|:-------------------|:----------------------------|:------------------------------------------------------------------|
+| **Critical** | 15 minutes         | 2 hours                    | Service unusable: unable to access the 121 platform               |
+| **High**     | 1 hour             | 4 hours                    | Service severely impaired: unable to send payments                |
+| **Medium**   | 2 hours            | 12 hours                   | Service partially impaired: issues with sending SMS               |
+| **Low**      | 3 hours            | In consultation with client | Service usable but with minor issues      |
+<!-- markdownlint-enable -->
+
+### After-Hours Support
+
+If you contact your **Account Manager** outside of regular business hours,
+your request will be forwarded to our support team at
+[support@121.global](mailto:support@121.global) and shared in our internal
+support alert system to ensure timely follow-up.
+
+________________________________________
+
+### Our Commitment to You
 
-- Level 1 First line support: Basic troubleshooting, Configuration issues, Login issues.
-- Level 2 Second line support: Escalation from Level 1, IM support, e.g. mass update on database and powerBI issues.
-- Level 3 Critical Development Team: Escalation from Level 2, Bug fixes.
+We aim at providing:
 
-!!! Warning "Emergency Responses Support hours"
-    During emergency responses, it may be required to broaden our support hours to evenings and weekends. This can be discussed with our teams, on a case by case basis.
+- **Personalized support** tailored to your needs
+- **Clear communication** with easy access to our team
+- **Fast resolution** of issues to minimize disruption
+Your feedback is important to us. If you have suggestions or need to discuss the
+ support provided, we’re always open to hearing from you and improving your experience.
 
-___
-Need further assistance? Contact your dedicated Account Manager or reach our [support team via email](mailto:support@121.global).
-___
+________________________________________
+Need further assistance? Contact your dedicated Account Manager or reach
+our [support team via email](mailto:support@121.global).
+________________________________________
```

Comment:
We could also create a "standard" block that can be included with "just a single line" anywhere. And then any change to the text only needs to happen in a single place.

Reply:
Theres the "snippets"-feature for this: https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown-extensions/?h=snippets#snippets

Reply:
@elwinschmitz I agree! I get an error on all formats I use. I am happy with a standard one

Reply:
@cassandradamico I've included a commonly used snippet for this piece of text. (see basically every last line of any page/file. ;) )

That should also solve some of the issues around "consistent use of the horizontal-line/separator thing"-syntax..